scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -404,6 +407,28 @@ def tag_exists(release_name):
         return False
 
 
+def add_token_remote():
+    """
+    Adds the Git remote to the given repository using
+    the provided installation (or personal) access token.
+    """
+    env = {}
+    env.update(os.environ)
+
+    print(f'git remote remove {REMOTE}')
+    subprocess.run(
+       ["git", "remote", "remove", REMOTE],
+       env=env,
+    )
+    print(f'git remote add {REMOTE} ~~REDACTED~~')
+    remote_url = f"https://x-access-token:{GITHUB_TOKEN}@github.com/{GITHUB_ORG}/{GITHUB_REPO}"
+    subprocess.run(
+       ["git", "remote", "add", REMOTE, remote_url],
+       env=env,
+       check=True,
```

Comment:
If this throws an exception, does that exception include the command that ran? That would leak the token.

Reply:
Great catch. I believe it does:
```
>>> subprocess.run(["false", "dddd"], check=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['false', 'dddd']' returned non-zero exit status 1.
```

I changed it to raise the Exception by us. PTAL

Reply:
stderr however does not:
```
$ git remote add asd https://github.com/pmtk/microshift.git
$ git remote add asd https://github.com/pmtk/microshift.git
error: remote asd already exists.
```

Reply:
That looks good, now. Thanks!